client/my-sites/plans-grid/style.scss

```diff
@@ -423,6 +423,7 @@ $mobile-card-max-width: 440px;
 .plan-features-2023-grid__plan-spotlight {
 	background-color: var(--studio-white);
 	.plan-features-2023-grid__plan-spotlight-card {
+		position: relative;
```

Comment:
I guess it could also be applied to the parent `plan-features-2023-grid__plan-spotlight` (same effect)

Reply:
As we talked, [I merged those classes](https://github.com/Automattic/wp-calypso/pull/83864/commits/992926516b3e5779bc02816094d721af132ea8d3) and removed the `...-card` one. Sending as soon as the tests pass.